src/services/anchor-service.ts

```diff
@@ -101,7 +101,7 @@ export default class AnchorService {
    * @returns whether or not an anchor was performed
    */
   public async anchorIfTooManyPendingRequests(): Promise<boolean> {
-    if (!config.merkleDepthLimit) {
+    if (config.merkleDepthLimit == 0 || config.merkleDepthLimit == undefined) {
```

Comment:
@stbrody I'm still not confident with this. If the merkleDepthLimit is set to '0' what should the behavior here be?

Reply:
well `'0' == 0`, so I think this should be fine?

Reply:
didnt realize!